include/mbgl/ios/MGLMapView.h

```diff
@@ -404,6 +404,30 @@ IB_DESIGNABLE
 - (void)setZoomLevel:(double)zoomLevel animated:(BOOL)animated;
 
 /**
+ * The minimum zoom level the can be shown at.
```

Comment:
how would you phrase this in the documentation? I'm having trouble coming up with a clear explanation